lms/djangoapps/instructor/enrollment.py

```diff
@@ -317,7 +317,11 @@ def _fire_score_changed_for_block(course_id, student, block, module_state_key):
             field_data_cache=cache,
             course_key=course_id
         )
-        points_earned, points_possible = weighted_score(0, module.max_score(), getattr(module, 'weight', None))
+        max_score = module.max_score()
+        if max_score is None:
```

Comment:
A few things:

- Right now, since grades aren't persisted, this is not an issue.  Viewing the progress page will always recompute the grades.
- Even once grades are persisted, it's an invariant that blocks with invalid `raw/weighted_possible` scores are not graded/scored.